src/tiles.rs

```diff
@@ -122,6 +122,19 @@ impl<Pane> Tiles<Pane> {
         self.tiles.values_mut()
     }
 
+    /// Get the pane instance for a given [`TileId`]
+    #[allow(dead_code)]
+    fn pane(&self, tile_id: &TileId) -> Option<&Pane> {
```

Comment:
This belong next to `get` as it is a similar thing

Reply:
Took the liberty to also fix the visibility of the function (should be public) and rename it to `get_pane`